.actor/actor.sh

```
@@ -283,153 +292,66 @@ DOCLING_API_ENDPOINT="http://localhost:5001/v1alpha/convert/source"
-    if [ -d "/tmp/actor-input/INPUT" ]; then
-        echo "Warning: /tmp/actor-input/INPUT exists as a directory. Removing it to create a file." >&2
-        rm -rf "/tmp/actor-input/INPUT"
-    fi
-
-    # Check multiple potential locations for input file
-    if [ -f "/tmp/actor-input/INPUT" ]; then
-        echo "Found standard Actor input file at /tmp/actor-input/INPUT" >&2
-        input=$(cat "/tmp/actor-input/INPUT")
-    elif [ -f "/input/INPUT" ]; then
-        echo "Found Actor input file at /input/INPUT" >&2
-        input=$(cat "/input/INPUT")
-
-    # Fallback to environment variable
-    elif [ -n "$APIFY_INPUT_JSON" ]; then
-        echo "Using APIFY_INPUT_JSON environment variable" >&2
-        input="$APIFY_INPUT_JSON"
-
-    # Last resort: use test input with md format
-    else
-        echo "No input found, using test input with md format" >&2
-        TEST_INPUT='{"documentUrl":"https://vancura.dev/assets/actor-test/facial-hairstyles-and-filtering-facepiece-respirators.pdf","ocr":true,"outputFormat":"md"}'
-        mkdir -p "/tmp/actor-input"
-        echo "$TEST_INPUT" > "/tmp/actor-input/INPUT"
-
-        # Read back the test input to ensure we get clean JSON
-        input=$(cat "/tmp/actor-input/INPUT")
-    fi
-
-    # Return only the JSON content
-    echo "$input"
-}
-
-# Get actor input
-INPUT=$(get_actor_input)
 echo "Input content:" >&2
 echo "$INPUT" >&2  # Send the raw input to stderr for debugging
 echo "$INPUT"      # Send the clean JSON to stdout for processing
 
-# Extract values from INPUT using Python
-echo "Using Python to parse input..."
-DOCUMENT_URL="$(echo "$INPUT" | python -c "import sys, json; print(json.load(sys.stdin).get('documentUrl', ''))")"
-OUTPUT_FORMAT="$(echo "$INPUT" | python -c "import sys, json; print(json.load(sys.stdin).get('outputFormat', 'md'))")"
-OCR_ENABLED="$(echo "$INPUT" | python -c "import sys, json; print(str(json.load(sys.stdin).get('ocr', True)).lower())")"
```

@netmilk Also, since the previous version had explicit validation for input parameters like `outputFormat` and `ocr`, that's now removed. If these parameters are supplied in the wrong format in the options object, there might be silent failures.